Add WireMock module to the registry (#3)
testcontainers/community-module-registry: modules/wiremock.md

---
title: WireMock
categories:

>   - cloud
>   - web
>   - other

Reviewer: I think the right would be
```suggestion
  - web
```

Author: As a suggested compromise, I will remove "other"

```suggestion
  - cloud
  - web
```

docs:
  - id: java
    url: https://github.com/wiremock/wiremock-testcontainers-java
    isThirdParty: true
    example: |
      ```java

>         import org.junit.jupiter.api.*;
>         import org.testcontainers.junit.jupiter.*;
>         import org.wiremock.integrations.testcontainers.testsupport.http.*;
> 
>         import static org.assertj.core.api.Assertions.assertThat;
> 
>         @Testcontainers
>         class WireMockContainerJunit5Test {
> 
>             @Container
>             WireMockContainer wiremockServer = new WireMockContainer("2.35.0")
>                     .withMapping("hello", WireMockContainerJunit5Test.class, "hello-world.json");
> 
>             @Test
>             void helloWorld() throws Exception {
>                 String url = wiremockServer.getUrl("/hello");
>                 HttpResponse response = new TestHttpClient().get(url);
>                 assertThat(response.getBody())
>                         .as("Wrong response body")
>                         .contains("Hello, world!");
>             }
>         }

Reviewer: Just to be consistent with the other examples

```suggestion
      WireMockContainer wiremockServer = new WireMockContainer("2.35.0")
                .withMapping("hello", WireMockContainerJunit5Test.class, "hello-world.json");
      wiremockServer.start();                
```

      ```
description: |

>     This module allows provisioning the WireMock server as a standalone container within your unit test,

Reviewer: ```suggestion
    This module allows provisioning the WireMock server as a standalone container within your tests,
```
